app/Models/HandleReservation.php

```diff
@@ -44,8 +44,17 @@ class HandleReservation extends ApiModel
         'reason' => 'sometimes|string|max:255',
     ];
 
-    public static function findAll(): Collection
+    public static function findAll(bool $activeOnly = false): Collection
     {
-        return HandleReservation::all()->sortBy('handle', SORT_NATURAL)->values();
+        $rows = HandleReservation::query();
```

Comment:
You can change HandleReservation::query() to be self::query(). The former is valid, although the code base uses self::query() within the models themselves.

Reply:
I think my IDE had a style warning when I initially tried that, since this is a static function but "self" implies you're calling an instance function. It feels a little wrong, but I guess I'll take consistency over correctness